src/components/dashboard/components/dictionary/DictionaryVersionsTable.jsx

```diff
@@ -12,12 +12,13 @@ const DictionaryVersionsTable = (version) => {
   const DATE_OPTIONS = {
     weekday: 'long', year: 'numeric', month: 'short', day: 'numeric',
   };
+  const traditionalOclUrl = 'https://qa.openconceptlab.org';
```

Comment:
Will this work for? https://openmrs.qa.openconceptlab.org/

Reply:
Yes it will. The traditional OCL is hosted under "https://qa.openconceptlab.org/" so that link won't change.

Reply:
The latter is a subdomain. The traditional OCL is on the main domain.